tests/Doctrine/Tests/Common/Cache/MemcachedCacheTest.php

```diff
@@ -49,6 +49,33 @@ public function testGetMemcachedReturnsInstanceOfMemcached()
         $this->assertInstanceOf('Memcached', $this->_getCacheDriver()->getMemcached());
     }
 
+    public function testDoContains()
+    {
+        $driver = $this->_getCacheDriver();
+        $reflection = new \ReflectionClass($driver);
+        $method = $reflection->getMethod('doContains');
```

Comment:
IMO, it would be better to write the test using the public API directly (i.e. ``contains``)